host.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "2.0",
+  "extensionBundle": {
```

Comment:
we usually prefer to control packages version though csjprog

https://github.com/pagopa/io-functions-services/blob/master/extensions.csproj